build/scripts/build-utils.ps1

```diff
@@ -330,7 +330,6 @@ function Restore-Packages([string]$msbuildDir = "", [string]$project = "") {
         $all = @(
             "Base Toolset:build\ToolsetPackages\BaseToolset.csproj",
             "Roslyn:Roslyn.sln",
-            "Samples:src\Samples\Samples.sln",
```

Comment:
Oh, woops, that was just a test to see how much it reduced restore time. I'll remove that.